scripts/codegen/CAN/App_CanMsgs.dbc

```diff
@@ -385,6 +668,23 @@ BA_ "GenMsgCycleTime" BO_ 126 1000;
 BA_ "GenMsgCycleTime" BO_ 127 1000;
 BA_ "GenMsgCycleTime" BO_ 128 1000;
 BA_ "GenMsgCycleTime" BO_ 129 1000;
+BA_ "GenMsgCycleTime" BO_ 160 100;
```

Comment:
How did we decide on these CAN msg frequencies?

Reply:
Those message cycle times were set by the inverter manufacturer. I don't think you can change them because it is the inverter that is broadcasting these messages. 

We have not decided yet on the actual torque request msg frequency but it is currently set at 5 ms by default.